docs/advanced/IncreaseDbSize.md

```diff
@@ -0,0 +1,25 @@
+# Increase AsyncStorage Database Size on Android
+
```

Comment:
I'd put here sections for platforms, i.e use H2 for `Android` and below `iOS` (which is not supported for now).